Feature/new blogposts (#268)
micro-ROS/micro-ROS.github.io: _posts/2020-12-16-ROS-In.md

---
title: micro-ROS @ ROS-Industrial Conference 2020! 
author: francesca-finocchiaro
---

micro-ROS was presented at the 2020 ROS-Industrial Conference in a joint talk given by Ralph Lange from Bosch, and by Francesca Finocchiaro and Pablo Garrido Sanchez from eProsima.

> In the first part of the talk, Ralph Lange discussed real-time support and determinism in both ROS 2 and micro-ROS, which is an important requirement for industrial automation. The talk also addressed the most recent developments added in the Foxy release for the Callback-group-level Executor and the roadmap for the Galactic release. Finally, he presented details on the rclc Executor for deeply embedded applications such as those addressed by micro-ROS.

Reviewer: ```suggestion
In the first part of the talk, Ralph Lange presented the ROS 2 Executor design and discussed latest developments towards real-time support and determinism, which are important requirements for industrial applications. The talk also addressed the most recent features added in the Foxy release and for the Galactic release. Finally, he presented details on the rclc Executor for deeply embedded applications such as those addressed by micro-ROS.
```

In the second part of the talk, Francesca Finocchiaro provided an overview of the micro-ROS project with a focus on its latest features and developments, and subsequently Pablo Garrido Sanchéz presented an entry-level hands-on demonstration intended for people willing to get started with micro-ROS.

<!-- Within this framework, we have delivered a talk in which we have provided an overview on the micro-ROS project with a special focus on its latest developments (Francesca Finocchiaro), and then presented an entry-level hands-on demonstration of its usage, intended for people willing to get started with micro-ROS (Pablo Garrido Sanchez).-->

> Find the presentations [here](/download/ROS-Industrial_workshop_ROS2.pdf) and [here](/download/ROSIn2020-FrancescaFinocchiaro.pdf).

Reviewer: ```suggestion
Find the presentations [here](/download/2020-12-16_Advanced_Execution_Management_with_ROS_2.pdf) and [here](/download/ROSIn2020-FrancescaFinocchiaro.pdf).
```

> <embed src="/download/ROS-Industrial_workshop_ROS2.pdf" type="application/pdf" width="100%" height="450px"/>

Reviewer: ```suggestion
<embed src="/download/2020-12-16_Advanced_Execution_Management_with_ROS_2.pdf" type="application/pdf" width="100%" height="450px"/>
```
